README.md

```diff
@@ -4,6 +4,7 @@
 [![TravisCI](https://travis-ci.com/nervosnetwork/ckb.svg?token=y9uR6ygmT3geQaMJ4jpJ&branch=develop)](https://travis-ci.com/nervosnetwork/ckb)
 [![Telegram Group](https://cdn.jsdelivr.net/gh/Patrolavia/telegram-badge@8fe3382b3fd3a1c533ba270e608035a27e430c2e/chat.svg)](https://t.me/nervosnetwork)
 [![Nervos Talk](https://img.shields.io/badge/discuss-on%20Nervos%20Talk-3CC68A.svg)](https://talk.nervos.org/)
+[![Codecov](https://codecov.io/gh/nervosnetwork/ckb/graph/badge.svg)](https://codecov.io/gh/nervosnetwork/ckb)
```

Comment:
Since we'll only run cov in master, it's better to use the master branch cov badge: `https://codecov.io/gh/nervosnetwork/ckb/branch/master/graph/badge.svg`

Reply:
Could you please also remove `[skip ci]` from the commit message? We have enforced that PR can only be merged when it passes the CI.

Reply:
updated